### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by tompizmor**
This is just a documentation PR. It adds the following sections to the readme:

- The requirements a chart source fulfill so that the tool can move it to another repository
- Changes performed in the chart code
- Harbor example
- Other improvements

### CODE REVIEW

**Comment by jotadrilo** [comment]
I don't see references to the new `repoName` field. Will you be adding those separately?

**Comment by tompizmor** [comment]
I want to merge first the other PR (#26), then rebase this and include pending info regarding `repoName`

File: README.md [hunk 1]
```diff
@@ -61,14 +61,205 @@ target:
...
+
+https://$HARBOR_DOMAIN/chartrepo/$HARBOR_PROJECT
+
+So if HARBOR_DOMAIN=my.harbor.com and HARBOR_PROJECT=my-project, you would need to specify this repo in the config file like:
+
+~~~yaml
+target:
+ repo:
+   kind: HARBOR
+   url: "https://my.harbor.com/chartrepo/my-project"
```
Comment (jotadrilo) [inline README.md:102]: do you require double quotes?

File: README.md [hunk 2]
```diff
@@ -61,14 +61,205 @@ target:
...
+~~~
+
+## Requirements
+
+In order for this tool to be able to successfully migrate a chart from a source repository to another it must fulfill the following requirements:
+
+- If the chart has dependencies they are specified in a requirements.yaml file. Currently, if the dependencies are specified in the Chart.yaml file the tool won't be able to manage them.
```
Comment (jotadrilo) [inline README.md:109]: what's the standard to mention files? Should we use `requirements.yaml` instead?

Comment (jotadrilo) [reply README.md:109]: Let's file an issue to solve managing deps from Chart.yaml files

File: README.md [hunk 3]
```diff
@@ -61,14 +61,205 @@ target:
...
+
+These files are updated with the new container registry where the chart should pull the images from.
+
+#### Update requirements.yaml and requirements.lock
+
+If the chart has any dependency, they should be registered in this file. The requirements.yaml and requirements.lock file will be updated to retrieve the dependencies from the target repository.
+
+#### Update README.md
+
+README files for bitnami charts includes a TLDR section with instructions to add the helm repository to helm cli and
```
Comment (jotadrilo) [inline README.md:141]: TODO?

File: README.md (hunk 1 above)
Comment (tompizmor) [reply README.md:102]: Probably not. I will check it

File: README.md [hunk 4]
```diff
@@ -61,14 +61,205 @@ target:
...
+    url: "https://charts.bitnami.com/bitnami"
+target:
+  containerRegistry: "my.registry.io"
+  containerRepository: "test"
+  repo:
+    kind: CHARTMUSEUM
+    url: "http://localhost:8080"
+~~~
+
+After executing the tool, these are the changes performed to the following files:
```
Comment (jotadrilo) [inline README.md:162]: can you add docs for devs so this example can be updated easily if we update the tool? (no need to do so here)

File: README.md (hunk 2 above)
Comment (tompizmor) [reply README.md:109]: Do you mean in markdown? I think they should use italics but not sure

File: README.md (hunk 3 above)
Comment (tompizmor) [reply README.md:141]: ups

File: README.md (hunk 4 above)
Comment (tompizmor) [reply README.md:162]: do you mean documenting how to get these diffs? Ook, I think I can document the process

File: README.md [hunk 5]
```diff
@@ -61,14 +61,205 @@ target:
     #   password: "PASSWORD"#
 ~~~
 
+> Note the `repo.url` property you need to specify is the same one you would use to add the repo to helm with the `helm repo add command`.
```
[the remark this answers is not in the document]
Comment (tompizmor) [reply README.md:65]: the first, although I omitted the `that`

File: README.md (hunk 2 above)
Comment (tompizmor) [reply README.md]: issue created: https://github.com/bitnami-labs/charts-syncer/issues/28

**Comment by jotadrilo** [comment]
LGTM!

**Pull request merged**